Backend/src/main/java/com/onbank/DatabaseConfig.java

```diff
@@ -0,0 +1,73 @@
+package com.onbank;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.dao.annotation.PersistenceExceptionTranslationPostProcessor;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.orm.jpa.JpaTransactionManager;
+import org.springframework.orm.jpa.JpaVendorAdapter;
+import org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean;
+import org.springframework.orm.jpa.vendor.HibernateJpaVendorAdapter;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import javax.persistence.EntityManagerFactory;
+import javax.sql.DataSource;
+import java.util.Properties;
+
+@EnableJpaRepositories
+@Configuration
+@EnableTransactionManagement
+public class DatabaseConfig {
+
+//    @Value()
+//    private final String costam;
```

Comment:
Tak jak mówiłem, powinniśmy wynieść wartości do propertiesów używając adnotacji @Value, podobnie jak to zakomentowane ;)